### PR TITLE
Add versionless Cask for Droplr.app

### PR DESCRIPTION
**Pull request by biasedbit**

### CODE REVIEW

File: Casks/droplr.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask 'droplr' do
+  version :latest
+  sha256 :no_check
+
+  # d.pr is the short url for droplr.com
```
Comment (vitorgalvao) [inline Casks/droplr.rb]: We have standard comments for cases like this ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5).

File: Casks/droplr.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  url 'https://d.pr/download/mac_direct_homebrew'
```
Comment (vitorgalvao) [inline Casks/droplr.rb]: Is this official at all? Where have you gotten this? We cannot accept it unless it is officially presented on the website.

Comment (biasedbit) [inline Casks/droplr.rb]: I was one of the original people that built Droplr.

Even though I'm no longer with the team, @levinunnink just set that redirect up, specifically for this cask. The direct download page is at https://d.pr/download/mac_direct, which does a JS redirect as opposed to the necessary HTTP redirect.

I've pinged Levi (Droplr's CEO) to comment on this thread.

Please elaborate on "unless it is officially presented on the website".

**Comment by levinunnink** [comment]
Hi Vito,

Yeah, it's official. We'll put it up on the website once it's approved. Just let me know if you need anything else from us.

Cheers!

**Comment by vitorgalvao** [comment]
@biasedbit could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

Thank you for confirming, @levinunnink. Please post here where on the website the link is, after you’ve done it.

**Comment by biasedbit** [comment]
> could you please squash your commits?

I will, as soon as I get confirmation this is good to go, as stated in the message of the last commit.

PS: I appreciate the link but I've been using git for a while now, I know my way around.

**Comment by vitorgalvao** [comment]
> I will, as soon as I get confirmation this is good to go

It is.

> PS: I appreciate the link but I've been using git for a while now, I know my way around.

TextExpander snippet; I just `cps`. That’s why I always include a link with that request.

**Comment by biasedbit** [comment]
Done.

> Please post here where on the website the link is, after you’ve done it.

This is a specific direct download URL for the cask and it's already up.

**Comment by vitorgalvao** [comment]
> This is a specific direct download URL for the cask and it's already up.

That’s not what I meant. To merge this cask, there needs to be something on the actual website (webpage) that links to this specific url, or we can’t merge it ([reasoning](https://github.com/caskroom/homebrew-cask/blob/7c010b42bd7ac679c1f002a0368b36bb3f495612/doc/CASK_LANGUAGE_REFERENCE.md#when-url-and-homepage-hostnames-differ-add-a-comment)). Alternatively, you can provide a non-shortened url (i.e. one that starts with `droplr.com` instead of `d.pr`).

Without either of those the cask cannot be accepted, as there’s no way for users to know it is trustworthy (this conversation isn’t enough, a user needs to be able to audit a cask for what it is, not have to search through issues to find something).

There’s still the third option of treating this as a [walled build](https://github.com/caskroom/homebrew-cask/blob/master/CONTRIBUTING.md#unofficial-vendorless-and-walled-builds) and submitting to [caskroom/unofficial](https://github.com/caskroom/homebrew-unofficial), but that’s not a great solution.

**Comment by vitorgalvao** [comment]
Also see https://github.com/caskroom/homebrew-cask/pull/13683, where this conversation happened before.

**Comment by levinunnink** [comment]
We can do that, Vítor. It'll just take a bit.

**Comment by levinunnink** [comment]
Just created https://droplr.com/homebrew. Does that meet the requirements?

**Comment by vitorgalvao** [comment]
> Just created https://droplr.com/homebrew. Does that meet the requirements?

Sure does.

**Comment by biasedbit** [comment]
Updated, tested, audited and rubocop'd.

**Comment by vitorgalvao** [comment]
Thank you for the changes and the fast response time. Merged.

**Pull request merged**